x-pack/plugins/reporting/public/management/components/report_info_flyout_content.tsx

```diff
@@ -186,7 +189,14 @@ export const ReportInfoFlyoutContent: FunctionComponent<Props> = ({ info }) => {
   ];
 
   const warnings = info.getWarnings();
-  const errored = info.getError();
+  const errored =
+    /*
+     * We link the user to documentation if they hit this error case. Note: this
+     * should only occur on cloud.
+     */
+    info.error_code === VisualReportingSoftDisabledError.code
+      ? i18nTexts.cloud.insufficientMemoryError('#')
```

Comment:
We need to update this to link to the cloud docs, once https://github.com/elastic/kibana/pull/129989 is merged